klt-cert-manager/controllers/keptnwebhookcontroller/config.go

```diff
@@ -2,25 +2,13 @@ package keptnwebhookcontroller
 
 import (
 	"time"
-
-	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 const (
 	// DeploymentName is the name used for the Deployment of any webhooks and WebhookConfiguration objects.
 	DeploymentName             = "klc-controller-manager"
```

Comment:
shall we go all in and align the naming?
```suggestion
	DeploymentName             = "klt-controller-manager"
```

Reply:
i'd say let's give it a shot :D